acceptance_tests/spec/gov_wifi_healtcheck_spec.rb

```diff
@@ -3,7 +3,7 @@
 
 describe 'GovWifi Healtcheck' do
   before do
-    db[:userdetails].insert(
+    user_details_db[:userdetails].insert(
```

Comment:
Just checking my understanding, 
so this new DB is called user-details and is fetching a table called user-details?

Reply:
Yeah, a bit repetitive but not sure how else to split this.

A database is essentially a table in this new setup